### PR TITLE
feat (argocd) minimal application + applicationset manifests

### PR DESCRIPTION
**Pull request by aajimal**
## Summary
This adds a few simplified and minimal manifests for ArgoCD `Applications` and `ApplicationSets` to this monorepo. 

The `Application` manifests are set to track this repository's main branch such that all changes to charts are synced automatically. 

The `ApplicationSet` manifests are set to automatically create `Application` instances for any pull requests that are open against this repository AND match a specific criteria (e.g. the appropriate labels). 

## Background
Since we had decided to move our `dev-cluster` charts into this monorepo for easier development flow it makes sense to also move some of our related configurations here as well. Furthermore, we can more easily script/template modifications to our app deployments with everything in one place 

## Changes
* Application for Rollup (tracking main)
* ApplicationSet for Rollup (tracking PRs)
* Application for Sequencer (tracking main)
* ApplicationSet for Sequencer (tracking PRs)
* Application for Sequencer-faucet (tracking main)
* ApplicationSet for Sequencer-faucet (tracking PRs)

## Testing
These were applied to the `argocd.dev.astria.org` environment.

### CODE REVIEW

File: charts/evm-rollup/templates/configmap.yaml [hunk 1]
```diff
@@ -104,7 +104,7 @@ metadata:
   namespace: {{ include "rollup.namespace" .  }}
 data:
   home_dir: "/home/geth"
-  data_dir: "/home/geth/{{ .Values.config.rollup.name }}geth"
```
Comment (aajimal) [inline charts/evm-rollup/templates/configmap.yaml:107]: This looked like a typo so I removed it.

File: charts/evm-rollup/files/scripts/init-geth.sh [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 set -o errexit -o nounset
 
+if [ ! -d $data_dir ] ; then
```
Comment (aajimal) [inline charts/evm-rollup/files/scripts/init-geth.sh:5]: Added this due to a problem starting container

**Comment by joroshiba** [comment]
This PR is stale because it has been open 45 days with no activity. Remove stale label or this PR will be
closed in 7 days.

**Comment by joroshiba** [comment]
This PR was closed because it has been stale.

**Pull request closed without merging**